charts/cht-chart-4x/templates/couchdb-n-deployment.yaml

```diff
@@ -25,8 +25,6 @@ spec:
         operator: {{ $root.Values.toleration.operator }}
         value: {{ $root.Values.toleration.value | quote }}
         effect: {{ $root.Values.toleration.effect }}
-      nodeSelector:
-        kubernetes.io/hostname: {{ index $root.Values.nodes (printf "node-%d" $nodeNumber) }}
```

Comment:
@nydr nodes selectors are needed in k3s when using mounted disks to different host machines. Since this one is already merged, I'll open a ticket to make them optional.

Reply:
Added a ticket [here](https://github.com/medic/helm-charts/issues/4).